src/Site_Command.php

```
@@ -579,7 +579,7 @@ public function create( $args, $assoc_args ) {
 
 		if ( $has_site_url ) {
 			$parsed_url = wp_parse_url( $assoc_args['site-url'] );
-			if ( ! isset( $parsed_url['host'] ) ) {
+			if ( false === $parsed_url || ! isset( $parsed_url['host'] ) ) {
```

PHPStan (2.2.6)

[error] Cannot access offset 'host' on mixed.


(offsetAccess.nonOffsetAccessible)